janusgraph-solr/src/main/java/org/janusgraph/diskstorage/solr/SolrIndex.java

```diff
@@ -950,6 +991,7 @@ public String mapKey2Field(String key, KeyInformation keyInfo) {
             switch (map) {
                 case TEXT: postfix = "_t"; break;
```

Comment:
I saw _t used someplace else above in this CR. Can you extract a string constant and reuse please?